mysql-test/suite/tokudb/t/fast_update_blobs.py

```diff
@@ -9,18 +9,23 @@ def main():
     print "source include/have_tokudb.inc;"
```

Comment:
The generated testfiles are still too large (as diff says +80K added lines) - please discuss with @georgelorchpercona to reduce the tests to as few SQL statements as possible. Even better if the python generators could be completely removed

Reply:
The tests check for the different combinations of data types. How it can be reduced? I could, for example, try to use loops inside of test case instead of python test generator. But loops in mtr are not well-designed, we can't iterate through arrays, for example. And, despite, I met some problems when I tried to use nested loops, I am not sure it is currently fixed. I could play with temporary tables or strings to emulate arrays, I could try to use pure SQL to implement enumeration, the but I am not sure the result code will be more readable then the initial python generator code.

Reply:
BTW, random string generator will look weird too on SQL.

Reply:
Apart from this each test has the comment which explains the test is generated, what means we should look at the test generator instead of trying to read the test text itself to understand what exactly is tested. The pattern of each test is the same - insert or change the data both in TokuDB and InnoDB tables, and compare their content, so the result file itself is not important for understanding if the test passed or failed, what is important is the result of tables content comparison. I could, for example, rewrite one of the tests using SQL, and the post here it's text to decide if it's reasonable to do the same with the rest of the tests.

Reply:
I have rewrote python generators in perl and embedded them into mtr tests.